Core/include/Acts/Navigation/MultiLayerSurfacesUpdater.hpp

```diff
@@ -45,15 +45,18 @@ class MultiLayerSurfacesUpdaterImpl : public INavigationDelegate {
   MultiLayerSurfacesUpdaterImpl(
       grid_type&& igrid, const std::array<BinningValue, grid_type::DIM>& icasts,
       const Transform3& itr = Transform3::Identity())
-      : grid(std::move(igrid)), casts(icasts), transform(itr) {}
+      : grid(std::move(igrid)), casts(icasts), transform(itr.inverse()) {}
```

Comment:
would it make sense to pass in the inverse from outside?

Reply:
yes, it could be like that!